test/setup.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+# ------------------------------------------------------------------
+# [korthout] Setup test repo to be used to test the backport action
+#
+# Produces a dir "repo" with history like:
+# 
+# (cd repo; git log --graph --oneline)
+# *-.   071b055 (HEAD -> master, release-2) Merge branches 'feature-b' and 'feature-c'
+# |\ \
+# | | * fa7b5cd (feature-c) chore: add to c
+# | | * 1b3090f feat: add c
+# | |/
+# |/|
+# * |   52241cf (release-1) Merge branch 'feature-a'
+# |\ \
+# | * | a908d73 (feature-a) feat: add a
+# |/ /
+# | * 651f859 (feature-b) chore: add to b
+# | * f072dc1 chore: add to b
+# | * fef5efe feat: add b
+# |/
+# * 922bcb9 init: add README.md
+# ------------------------------------------------------------------
+
+set -e  # Abort script at first error, when a command exits with non-zero status (except in until or while loops, if-tests, list constructs)
+# set -u  # Attempt to use undefined variable outputs error message, and forces an exit
```

Comment:
do we need these comments? Are they for debugging?

Reply:
Yes, these can be handy while developing. They were provided by the template I used